ci_framework/roles/build_containers/tasks/main.yml

```diff
@@ -35,17 +35,22 @@
 
 - name: Run container_build.sh script
   become: true
+  when: not cifmw_build_containers_run_hotfix | bool
   args:
     chdir: "{{ cifmw_build_containers_basedir }}/artifacts"
   ansible.builtin.shell: |
     set -o pipefail && bash build_containers.sh 2>&1 {{ cifmw_build_containers_timestamper_cmd }} > {{ cifmw_build_containers_basedir }}/logs/build.log
   environment:
     PATH: /usr/local/bin:{{ ansible_env.PATH }}
 
+- name: Include tasks from hotfix.yml
+  when: cifmw_build_containers_run_hotfix | bool
+  ansible.builtin.include_tasks: hotfix.yml
+
 - name: Make sure authfile exists
   when:
     - cifmw_build_containers_authfile_path is defined
-    - cifmw_build_containers_push_containers | bool
+    - cifmw_build_containers_run_hotfix | bool
```

Comment:
We need to run this task for push_containers. Why we are changing the conditional?